graphics/macgui/macmenu.cpp

```diff
@@ -312,6 +312,9 @@ int MacMenu::numberOfMenus() {
 MacMenuItem *MacMenu::getMenuItem(const Common::String &menuId) {
 	MacMenuItem *menu = nullptr;
 
+	if (!this)
```

Comment:
`this` never can be NULL in instantiated objects. This check will never be true.